example/.gitattributes

```diff
@@ -0,0 +1 @@
+src/subdir/ignored.js rules_lint_ignored=true
```

Comment:
could we also add a line like
`pnpm-lock.yaml linguist-generated=true`
and assert that we don't try to format generated code?